source/d3d11/draw_call_tracker.hpp

```diff
@@ -9,32 +9,43 @@ namespace reshade::d3d11
 	class draw_call_tracker
 	{
 	public:
-		UINT vertices() const { return _counters.vertices; }
-		UINT drawcalls() const { return _counters.drawcalls; }
+		struct draw_counter {
```

Comment:
Maybe rename this to something like `draw_info` or `draw_data` or `draw_stats`, since it's just data and doesn't contain any functionality that would warrant the `counter` name. Also, extra points for style: =)
```c++
struct draw_stats
{
};
```